focus-ios/Blockzilla/Modules/WebView/LegacyWebViewController.swift

```diff
@@ -517,15 +519,15 @@ extension WebViewController: WKNavigationDelegate {
     }
 }
 
-extension WebViewController: BrowserState {
+extension LegacyWebViewController: BrowserState {
```

Comment:
I think it could be worthwhile to rename BrowserState to LegacyBrowserState as well as I don't think it will be used with the new engine code. What do you think?
```suggestion
extension LegacyWebViewController: LegacyBrowserState {
```

Reply:
Makes sense, I've updated this